commands.go

```diff
@@ -109,8 +109,8 @@ type Cmdable interface {
 	HLen(key string) *IntCmd
 	HMGet(key string, fields ...string) *SliceCmd
 	HMSet(key string, fields map[string]string) *StatusCmd
-	HSet(key, field, value string) *BoolCmd
-	HSetNX(key, field, value string) *BoolCmd
+	HSet(key, field, value interface{}) *BoolCmd
```

Comment:
Can you please leave key, field to be a string? This is consistent with other commands.